app/Hutch.Relay/Controllers/TaskController.cs

```diff
@@ -69,6 +72,18 @@ public async Task<IActionResult> Result(Guid uuid, string collectionId, [FromBod
     {
       //Aggregate SubTasks Result.Count
       var finalResult = await resultsService.AggregateResults(subtask.RelayTask.Id);
+      // Apply Obfuscation functions if configured
```

Comment:
oh good catch. Yeah centralising the logic in the `ObfuscationService` makes sense, and then just calling it simply from both locations, much like we do with `AggregateResults()`.

This also resolves my other design thought which was around having `ObfuscationService` read the config, rather than here.